src/store/user.js

```diff
@@ -142,7 +147,7 @@ const actions = {
     }
     const token = vueAuthInstance.getToken()
     if (token) {
-      init(this._vm.$client, token)
+      await init(this._vm.$client, token)
```

Comment:
The `await` here should not be needed, as it's the last statement within the function.